bentoml/_internal/frameworks/tensorflow_v2.py

```diff
@@ -365,19 +156,30 @@ def __call__(self, inputs):
 
         # then save the given model to BentoML modelstore:
         model = NativeModel()
-        tag = bentoml.tensorflow.save("native_toy", model)
+        tag = bentoml.tensorflow.save_model("native_toy", model)
 
     .. note::
 
-       :code:`bentoml.tensorflow.save` API also support saving `RaggedTensor <https://www.tensorflow.org/guide/ragged_tensor>`_ model and Keras model. If you choose to save a Keras model
-       with :code:`bentoml.tensorflow.save`, then the model will be saved under a :obj:`SavedModel` format instead of :obj:`.h5`.
+       :code:`bentoml.tensorflow.save_model` API also support saving `RaggedTensor <https://www.tensorflow.org/guide/ragged_tensor>`_ model and Keras model. If you choose to save a Keras model
+       with :code:`bentoml.tensorflow.save_model`, then the model will be saved under a :obj:`SavedModel` format instead of :obj:`.h5`.
 
     """  # noqa
-    context: t.Dict[str, t.Any] = {
-        "framework_name": "tensorflow",
-        "pip_dependencies": [f"tensorflow=={get_tf_version()}"],
-        "import_from_tfhub": False,
-    }
+    context = ModelContext(
+        framework_name="tensorflow",
+        framework_versions={"tensorflow": get_tf_version()},
+    )
+
+    # will add signatures inference from tf_signatures later
+    if signatures is None:
+        logger.info(
+            "Using the default model signature for TensorFlow v2 ({signatures}) for model {name}."
+        )
+        signatures = {
+            "__call__": ModelSignature(
+                batchable=False,
+            )
+        }
+
```

Comment:
```suggestion
        signatures = {
            "__call__": ModelSignature(
                batchable=False,
            )
        }
        logger.info(
            "Using the default model signature {signatures} for TensorFlow models."
        )
```

Reply:
I think printing `"{'__call__': {'batchable': False}}"` is slightly nicer than printing `"{'__call__': ModelSignature(batchable=False, batch_dim=0, input_spec=None, output_spec=None)}"` here, what do you think?

Reply:
done